ResearchKit/Common/ORKDefines.h

```diff
@@ -103,3 +103,15 @@ typedef NS_ENUM(NSInteger, ORKAudioChannel) {
     ORKAudioChannelRight
 } ORK_ENUM_AVAILABLE;
 
+
+/**
+ Body side constants.
+ */
+typedef NS_ENUM(NSInteger, ORKSide) {
```

Comment:
I kind of like `ORKBodySagittalLeft` and `ORKBodySagittalRight` from @DanKeen, what do you think?